TUTORIAL.md

```diff
@@ -0,0 +1,57 @@
+# Guide to Solving Countdown to Midnight
+
+The first error message we are getting after running `rspec` is: 
+
+### `#countdown`
+
+```bash
+LOOPS
+  #countdown
+    outputs '<n> SECOND(S)!' string to STDOUT for each count (FAILED - 1)
+    returns HAPPY NEW YEAR! (FAILED - 2)
+  #countdown_with_sleep
+    should take at least 5 seconds to execute (FAILED - 3)
+
+Failures:
+
+  1) LOOPS #countdown outputs '<n> SECOND(S)!' string to STDOUT for each count
+     Failure/Error: expect { countdown(10) }.to output(countdown_output).to_stdout
+     ArgumentError:
+       wrong number of arguments (1 for 0)
+```
+
+`Wrong number of arguments (1 for 0)` means our countdown method expects from us that we pass in an argument.
+
+```ruby
+def countdown(seconds_to_midnight)
+end
+```
+
+Now after passing in the seconds to midnight our method should use a `while-loop` to check if the seconds we passed in are greater than 0, as long as that condition is true, it should countdown the seconds and decrement them every time the loop runs until 0. When we hit 0 it should say `"HAPPY NEW YEAR"`.
```

Comment:
Hmmm this is a big jump. Do you know want to maybe give another `rspec` run and show that output? Or maybe show the tests themselves and say "ok so I know if the input is 2, my method needs to puts 2 times and then return Happy New Year. Let's first get it to print out the "N Seconds" stuff first. PAssed another test! now let's get the return right."